frontend/src/App.tsx

```diff
@@ -6,26 +6,26 @@ import AddComponentPage from "./pages/AddComponentPage";
 
 function App() {
 
-  const {addComponent} = useComponent();
+    const {addComponent, getAllComponents, getComponentById, components} = useComponent();
 
 
-  return (
-    <div className="App">
-      <h1>Check&Computify</h1>
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-        </p>
-      </header>
-      <AddComponentPage addComponent={addComponent} component={{
-        id: undefined,
-        name: '',
-        category: '',
-        combinationCode: '',
-        score: ''
-      }}/>
-    </div>
-  );
+    return (
+        <div className="App">
+            <h1>Check&Computify</h1>
+            <header className="App-header">
+                <img src={logo} className="App-logo" alt="logo"/>
+                <p>
+                </p>
```

Comment:
wozu ist dieser Tag da?

Reply:
Ist ein placeholder für mich, damit ich weiß wo was hingehört später.
Macht aber so in der Form im Projekt keinen Sinn. 
Gut gesehen und thx ! :)